app/static/lib/conp-react/src/DatasetElement/DownloadModalWindowElement/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import * as ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+
+const DownloadModalWindowElement = (props) => {
+  const { authorized, imagePath, ...element } = props;
+
+  const close = (event) => {
+    $("#downloadModal").modal("hide");
+  };
+
+  return ReactDOM.createPortal(
+    <div className="modal-content">
+      <div className="modal-body">
+        <table>
```

Comment:
I would rather use a bootstrap container for laying out the CONP image and warning text if we could.